src/main/java/com/google/sps/scheduler/Scheduler.java

```diff
@@ -24,14 +25,19 @@
 import java.time.LocalDate;
 import java.time.LocalTime;
 import java.util.ArrayList;
+import java.util.Comparator;
 import java.util.Iterator;
 import java.util.List;
 import java.util.Set;
 import java.util.TreeSet;
-import java.util.Comparator;
 import java.util.concurrent.TimeUnit;
 
-/** Class that allows to schedule some tasks in the free time slot of the calendar. */
+import static converter.TimeConverter.epochInMilliseconds;
+import static converter.TimeConverter.epochToDateTime;
+
+/**
+ * Class that allows to schedule some tasks in the free time slot of the calendar.
```

Comment:
Don't start with "Class that...", just write "allows to ..."
https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment